Physics/Dynamics/ProductionFormFactor.hpp

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) 2013, 2019 The ComPWA Team.
+// This file is part of the ComPWA framework, check
+// https://github.com/ComPWA/ComPWA/license.txt for details.
+
+#ifndef PHYSICS_DYNAMICS_PRODUCTION_FORMFACTOR_HPP_
+#define PHYSICS_DYNAMICS_PRODUCTION_FORMFACTOR_HPP_
+
+#include <boost/property_tree/ptree.hpp>
```

Comment:
This class should not depend on the boost property tree. All of the property tree code should go into the IntensityBuilderXML file

Reply:
Yes, you are right.